warehouse/templates/pages/stats.html

```diff
@@ -40,7 +40,7 @@ <h2>{% trans %}Top storage users{% endtrans %}</h2>
       <thead>
         <tr>
           <th scope="col">{% trans %}Project name{% endtrans %}</th>
-          <th scope="col">{% trans %}Sum of release files (bytes){% endtrans %}</th>
+          <th scope="col">{% trans %}Sum of release files (bytes){% endtrans %} ↓</th>
```

Comment:
Nit: I'm not sure about this additional `↓`, it might convey to the user that you can click on it to reverse the order.

Reply:
Agreed!

Reply:
True, but I think it's preferable to no indicator of sort order... other ideas?

Reply:
Personally I think the copy above is quite clear:

> Top storage users
> Here is a list of the top 100 packages based on sum of their packages sizes (in bytes). 

If the table was presented on its own I would agree with you 🙂

Reply:
Who reads copy? :) Will remove.